src/main/scala/com/amazon/deequ/analyzers/applicability/Applicability.scala

```diff
@@ -140,7 +141,14 @@ private[deequ] object Applicability {
     if (shouldBeNull(nullable)) {
       null
     } else {
-      new Timestamp(Random.nextLong())
+      val Low = 100
```

Comment:
How is this change related to the Spark/Scala update?

Reply:
This was a requirement to get spark 3.0 to work. It throws a long overflow exception in the checks.ApplicabilityTest.scala test when working with spark 3.0  I changed this code to operate in a valid timestamp range and it works after that. I tested with spark 2.x and it also passes the tests.